### PR TITLE
Merge server raws before save the local item.

### PR DESCRIPTION
**Pull request by beshkenadze**
Fix import files.

Signed-off-by: Aleksandr Beshkenadze <ab@caesar.team>

### CODE REVIEW

**Review by sa715 — APPROVED** [review]

File: packages/components/ItemFields/view/Attachments.js [hunk 1]
```diff
@@ -108,21 +62,22 @@ export const Attachments = ({
 
   // TODO: Add loader if raws are not ready
   const handleClickDownloadFile = attachment => {
-    const { name, ext } = attachment;
-    const raw = itemRaws[attachment.id];
+    dispatch(downloadItemAttachment({ itemId, attachment }));
+    // const { name, ext } = attachment;
```
Comment (cu4xxi) [inline packages/components/ItemFields/view/Attachments.js:66]: lets remove redundant comments

File: packages/components/ItemFields/view/Attachments.js [hunk 2]
```diff
@@ -108,21 +62,22 @@ export const Attachments = ({
...
 
   const handleClickDownloadAll = () => {
-    const files = itemAttachments.map(attachment => ({
-      raw: raws[attachment.id],
-      name: `${attachment.name}.${attachment.ext}`,
-    }));
-
-    downloadAsZip(files);
+    dispatch(downloadItemAttachments({ itemId }));
+    // const files = itemAttachments.map(attachment => ({
```
Comment (cu4xxi) [inline packages/components/ItemFields/view/Attachments.js:76]: and here too :)

File: packages/containers/Preferences/Preferences.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            onClick={handleClearCache}
+            icon="update"
+            color="black"
+          >
+            Reset Application Cache
+          </ResetCacheButton>
+        </>
+      }
+    >
+      <>¯\_(ツ)_/¯</>
```
Comment (cu4xxi) [inline packages/containers/Preferences/Preferences.js:40]: ai-ai-ai

File: packages/containers/Preferences/Preferences.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { Button, SettingsWrapper } from '@caesar/components';
+import styled from 'styled-components';
```
Comment (cu4xxi) [inline packages/containers/Preferences/Preferences.js:4]: Let's first import third-party libraries, and then our components/functions

**Review by cu4xxi — APPROVED** [review]

File: packages/containers/Preferences/Preferences.js (hunk 3 above)
Comment (beshkenadze) [reply packages/containers/Preferences/Preferences.js:40]: ¯\_(ツ)_/¯

**Pull request merged**